app/views/public/credits.html.haml

```diff
@@ -1,59 +1,4 @@
 - content_for :title, 'Credits'
 - menu_default_link('credits')
 
-%p
-  The development of this web application was an initiative of the
-  %a{href: 'https://www.qut.edu.au/'} Queensland University of Technology's
-  %a{href: 'http://research.ecosounds.org/'} Ecoacoustics Research Group.
-  This website makes use of a range of other technologies and libraries.
-
-%p
-  More information can be found on the
-  %a{href: 'https://github.com/QutBioacoustics/baw-server'} Github project page.
-
-%h2
-  Programs and libraries
-
-%ul
-  %li
-    %a{href: 'http://rubyonrails.org/'} Ruby on Rails
-    (with a number of additional gems)
-  %li
-    %a{href: 'https://github.com/resque/resque'} Resque
-  %li
-    %a{href: 'http://redis.io'} Redis
-  %li
-    Command line audio tools:
-    %ul
-      %li
-        %a{href: 'http://www.ffmpeg.org/'} ffmpeg
-        (for audio conversion and gathering audio file information)
-      %li
-        %a{href: 'http://sox.sourceforge.net/'} SoX
-        (to create spectrograms and resample audio)
-      %li
-        %a{href: 'http://www.wavpack.com/'} WavPack
-        (to expand compressed .wv files)
-      %li
-        %a{href: 'http://mp3splt.sourceforge.net/mp3splt_page/home.php'} mp3split
-        (for quickly segmenting large .mp3 files)
-
-%ul
-  %li
-    %a{href: 'https://angularjs.org/'} AngularJS
-  %li
-    %a{href: 'https://d3js.org/'} D3.js
-%h2
-  Platforms and services
-
-%ul
-  %li
-    %a{href: 'https://github.com'} Github
-  %li
-    %a{href: 'https://travis-ci.org'} TravisCI
-
-- if partial_exists?('public/credits')
-  %h1
-    = Settings.organisation_names.parent_site_name
-    Credits
-  = render partial: 'public/credits'
\ No newline at end of file
+= raw (controller.render cms_page: '/credits').gsub(/<h1>.*<\/h1>/, '')
```

Comment:
Temporary hack. The view system currently sticks its own `<h1>` in and thus two headings (one from rails, one from CMS) is redundant. Since we're deprecated all the rails views, this is a quick, and dirty, wasy to remove the other `<h1>`.